networkd/network_server.ml

```diff
@@ -317,6 +317,14 @@ module Interface = struct
 			Ethtool.set_offload name params
 		) ()
 
+	let is_fcoe_supported _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			let output = Dcbtool.is_fcoe_supported name in
+			if String.has_substr output "Successful"
```

Comment:
Why don't you do this part in Dcbtool.is_fcoe_supported, making that function return a bool instead of a string?